assets/js/components/DashboardNavigation.js

```diff
@@ -160,10 +132,13 @@ export default function DashboardNavigation() {
 		if ( global.location.hash !== '' ) {
 			setTimeout( () => {
 				global.scrollTo( {
-					top: getContextScrollTop(
-						global.location.hash.substr( 1 ),
-						breakpoint
-					),
+					top:
+						global.location.hash !== ANCHOR_ID_TRAFFIC
```

Comment:
The location hash contains `#` at the beginning, so it will never be equal `ANCHOR_ID_TRAFFIC`.
```suggestion
						global.location.hash.substr( 1 ) !== ANCHOR_ID_TRAFFIC
```

Reply:
Probably worth assigning the hash to a variable and then use it in the condition and in the function call:

```js
const hash = global.location.hash.substr( 1 );
global.scrollTo( {
    top: hash !== ANCHOR_ID_TRAFFIC ? getContextScrollTop( hash, breakpoint ) : 0
    ...
} );

Reply:
Apologies I missed it. Good catch 👍